elixir/apps/domain/priv/repo/migrations/20250611232514_remove_membership_rules.exs

```diff
@@ -0,0 +1,15 @@
+defmodule Domain.Repo.Migrations.RemoveMembershipRules do
+  use Ecto.Migration
+
+  def up do
+    alter table(:actor_groups) do
```

Comment:
Are we going to have lock issues here?  Won't this try to lock the `actor_groups` table and isn't that possibly going to interfere with our directory sync jobs?

Reply:
So basically anything we do to auth_providers is at risk, but other tables should be fine.